tests/unit/test_cli.py

```diff
@@ -542,3 +544,9 @@ def test_migrate_dbsql_dashboards(ws, caplog):
 def test_revert_dbsql_dashboards(ws, caplog):
     revert_dbsql_dashboards(ws)
     ws.dashboards.list.assert_called_once()
+
+
+def test_lint_local_code(ws):
+    path_to_scan = Path(Path(__file__).parent, "source_code", "samples")
+    problems = lint_local_code(ws, path=path_to_scan)
+    assert len(problems) > 0
```

Comment:
how long does it take to lint UCX codebase via `databricks labs ucx lint-local` command invocation? please add a second into README.md for users to adopt it.

Reply:
let's aim for unit tests to execute as fast as possible - there's still room for `@retried` optimization . the longest running test would slow down the entire suite. move this test to integration tests